bigquery/snippets/snippet.go

```diff
@@ -405,6 +470,121 @@ func queryLegacy(client *bigquery.Client, sqlString string) error {
 	return nil
 }
 
+func queryLegacyLargeResults(client *bigquery.Client, dstDatasetID, dstTableID string) error {
+	ctx := context.Background()
+	// [START bigquery_query_legacy]
```

Comment:
Don't we have a different region tag for "large results"? `bigquery_query_legacy_large_results`

https://github.com/GoogleCloudPlatform/google-cloud-java/blob/c59f31dc447ee5aa97f8e7b4e677af4d06aaa4fa/google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/CloudSnippets.java#L116

Note: this will concatenate to the `queryLegacy` sample above, so I presume you did mean to use a different region tag.